mockwebserver/src/main/java/okhttp3/mockwebserver/MockResponse.java

```diff
@@ -205,6 +206,15 @@ public MockResponse setSocketPolicy(SocketPolicy socketPolicy) {
     return this;
   }
 
+  public int getHttp2ErrorCode() {
+    return http2ErrorCode;
+  }
+
+  public MockResponse setHttp2ErrorCode(int http2ErrorCode) {
```

Comment:
+Javadoc explaining that this is used only for `SocketPolicy. RESET_STREAM_AT_START`